.travis.yml

```diff
@@ -114,9 +114,9 @@ script:
 matrix:
   include:
     - php: 5.6
-      env: "DB=mysql CORE_BRANCH=stable12 TEST_JS=TRUE"
+      env: "DB=mysql CORE_BRANCH=master TEST_JS=TRUE"
```

Comment:
Should it not be stable13 now that we branched off?

Reply:
See line 25